frontend/src/Routes.test.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import { MemoryRouter } from "react-router-dom";
+import { configure, mount } from "enzyme";
+import Adapter from 'enzyme-adapter-react-16';
+import AlertConfiguration from './AlertConfiguration/AlertConfigurationComponents/AlertConfiguration';
+import AlertsManagement from './AlertsManagement/AlertsManagement';
+import App from './App';
+import ErrorPage from './ErrorPage';
+import Routes from "./Routes";
+
+configure({ adapter: new Adapter() });
+
+describe("full app routing", () => {
+  it("should display App component on home route", () => {
+    const wrapper = mount(
+      <MemoryRouter initialEntries={[ '/' ]} initialIndex={0}>
+        <Routes />
+      </MemoryRouter>
+    );
+
+    expect(wrapper.find(App)).toHaveLength(1);
```

Comment:
just to make sure `wrapper.find(App)` makes sure `App` is rendered once?

Reply:
yup!